README.md

```diff
@@ -131,7 +133,6 @@ Check out the **[contributing wiki](https://github.com/pion/webrtc/wiki/Contribu
 * [Stefan Tatschner](https://rumpelsepp.org/contact.html)
 * [Hayden James](https://github.com/hjames9)
 * [Jozef Kralik](https://github.com/jkralik)
-* [Robert Eperjesi](https://github.com/epes)
```

Comment:
I used `git revert`, sorry.

Reply:
If this is about to be merged, I will remove this line from the commit.

Reply:
Please remove it now, no need to wait.

Reply:
done.